app/grandchallenge/archives/views.py

```diff
@@ -368,12 +371,12 @@ class ArchiveEditArchiveItem(
     form_class = ArchiveItemForm
     template_name = "archives/archive_item_form.html"
     permission_required = (
-        f"{Archive._meta.app_label}.upload_{Archive._meta.model_name}"
+        f"{ArchiveItem._meta.app_label}.change_{ArchiveItem._meta.model_name}"
     )
     raise_exception = True
 
     def get_permission_object(self):
-        return self.archive
+        return get_object_or_404(ArchiveItem, pk=self.kwargs["pk"])
```

Comment:
This method can be removed now, right?

Reply:
For this view, we still need it. It's a FormView and it does not have a get_object() method defined, so either we define `object` on the view, or we add a `get_object()` method to the view, or (in this is how I've seen it done in other places in the code base) we just overwrite `get_permission_object()`. I think the latter is the easiest to understand, right?

Reply:
OK I see, then it's best to use `self.archive_item` as the handling is already done there.

Reply:
Ah right, that makes sense! Thanks!